src/app/Main/Body/QueryPage/Results/Results-Table.component.tsx

```diff
@@ -82,7 +79,6 @@ export function ResultsTableComponent(props: ResultTableProps): JSX.Element {
                             {RowValues(
                                 row,
                                 classes,
-                                props.isIsolates,
                                 props.isRowAndCol,
                                 tabelCellStyle

```

Comment:
Avoid using callbacks as props. React is up -> down not down -> up the hierarchy. By using callbacks you could easily break all kinds of stuff and introduce side effects to your code which could become uncontrollable in a later state of the app.